openshift/templates/rails-postgresql-persistent.json

```diff
@@ -408,7 +408,7 @@
         "annotations": {
           "description": "Defines how to deploy the database",
           "template.alpha.openshift.io/wait-for-ready": "true",
-          "image.openshift.io/triggers": "[{\"from\":{\"kind\":\"ImageStreamTag\",\"name\":\"postgresql:${POSTGRESQL_VERSION}\"},\"fieldPath\": \"spec.template.spec.containers[0].image\"}]"
+          "image.openshift.io/triggers": "[{\"from\":{\"kind\":\"ImageStreamTag\",\"name\":\"postgresql:${POSTGRESQL_VERSION}\", \"namespace\": \"openshift\"},\"fieldPath\": \"spec.template.spec.containers[0].image\"}]"
```

Comment:
This is not needed. Before testing you are already in required 'namespace'

Reply:
Or use variable https://github.com/sclorg/rails-ex/blob/master/openshift/templates/rails-postgresql-persistent.json#L522, that is better approach